aws/resource_aws_iam_user_login_profile.go

```diff
@@ -19,30 +19,38 @@ func resourceAwsIamUserLoginProfile() *schema.Resource {
 	return &schema.Resource{
 		Create: resourceAwsIamUserLoginProfileCreate,
 		Read:   schema.Noop,
-		Update: schema.Noop,
+		Update: resourceAwsIamUserLoginProfileUpdate,
 		Delete: schema.RemoveFromState,
```

Comment:
I think we can also handle the Delete in another request: if you try to create a login profile for a user that has already been created, you will get a ConflictException. When we don't want to keep the login profile, we don't want only to remove it from the state: we actually want to delete it, to avoid conflicts.